app/locale/nl-NL.coffee

```diff
@@ -52,7 +52,7 @@ module.exports = nativeDescription: "Nederlands (Nederland)", englishDescription
 
   play:
     play_as: "Speel als " # Ladder page
-#    compete: "Compete!" # Course details page
+    compete: "Compleet!" # Course details page //in English it says 'complete', so translated that instead.
```

Comment:
Alright, then it should be 'ga de strijd aan'. I'll correct it and proceed translating.